region/brinstar/red/Red Tower.json

```diff
@@ -761,6 +859,11 @@
       "name": "Leave with Extended Moondance",
       "requires": [
         {"obstaclesNotCleared": ["A"]},
+        {"or": [
+          "HiJump",
```

Comment:
curious on this. I had a very hard time getting a Beetom up here with just wall jumps - it would knock the Beetom off. I'm assuming you might want A not cleared. It may be nice to have another node at the top of the room with a Beetom. It could be helpful to reduce the duplication and inconsistencies here and with g-mode setups, and allow for a more detailed description.

Reply:
In this scenario we also have Ice to freeze the Rippers, so the wall jumps are only needed to get up the big gaps before the Rippers. I tested it now and while the Beetom can be annoying and fall off, it is possible to get it to the top.